content/applications/websites/ecommerce/products.rst

```diff
@@ -4,266 +4,341 @@
 Products
 ========
 
-Odoo allows you to create, import, and manage your products' pages all within the **Website** app.
+**Odoo eCommerce** allows you to :ref:`create <products/create-products>`,
+:ref:`import <products/import-products>`, and manage your :ref:`product pages
+<products/product-pages>` directly within the Website app. You can also add
+:ref:`product variants <products/product-variants>` and
+:ref:`digital files <products/digital-files>`, :ref:`translate <products/translation>`
+your product page content, :ref:`manage your stock <products/stock-management>`, and enable
+:ref:`product comparisons <products/product-comparison>`.
 
-Add products to the catalog
-===========================
+Add products
+============
 
-To add a product to your catalog, you can either do it in:
+.. _products/create-products:
 
-- From anywhere on your website, click :menuselection:`+ New --> Product`. Enter the name of your
-  product, and :guilabel:`Save`;
-- :menuselection:`Website --> eCommerce --> Products --> Create`;
-- or by :ref:`importing data <import-data>` using XLSX or CSV files. To do so, go to
-  :menuselection:`Website --> eCommerce --> Products`. Click on :guilabel:`Favorites` and
-  :ref:`Import records <import-data>`.
+Create products
+---------------
 
-.. seealso::
-   - :doc:`products/catalog`
-   - :doc:`../../sales/sales/products_prices/products/import`
-   - :doc:`Product-related documentation <../../sales/sales>`
+From anywhere on your website, click :guilabel:`+ New`, then :guilabel:`Product`. Enter the
+:guilabel:`Product Name`, :guilabel:`Sales Price`, the default :guilabel:`Customer Taxes` for
+local transactions, and :guilabel:`Save`. You can then update the product details, add an image,
+and :ref:`customize <products/customization>` the product page. When you :guilabel:`Save`, the
+product page is automatically published.
```

Comment:
Someone who wants to create a product without making it immediately available will probably not find the information contained in this list of tips easy to find. Information on publishing a page can be found here : https://www.odoo.com/documentation/17.0/applications/websites/website/pages.html?highlight=publish#website-un-publish-page. I added a ref to that doc, to avoid duplicating the content, hope this makes sense to you :)